plugins.json

```
@@ -88,6 +88,20 @@
 			"author": "Avery",
 			"minCrankshaftVersion": "0.2.1",
 			"source": "https://git.sr.ht/~avery/custom-library-tabs"
+		},
+		{
+			"id": "steamdeckhq-crankshaft-plugin",
+			"repo": "https://github.com/shelbykleindesign/steamdeckhq-crankshaft-plugin",
+
+			"version": "0.2.0",
+			"archive": "https://github.com/shelbykleindesign/steamdeckhq-crankshaft-plugin/blob/main/download/0.2.0/steamdeckhq-crankshaft-plugin.tar.gz",
```

This is linking to the Github page for this file, instead of the file itself, looks like this is the direct file link:

```suggestion
			"archive": "https://github.com/shelbykleindesign/steamdeckhq-crankshaft-plugin/raw/main/download/0.2.0/steamdeckhq-crankshaft-plugin.tar.gz",
```